### PR TITLE
Some code cleanups

### PR DESCRIPTION
**Pull request by oscargus**
While I was at it: some code cleanups...

* entrySet instead of keySet
* correct order of static and final
* used equalsIgnoreCase instead of equals() and toLowerCase()
* added default switch case
* "correct" form of lambda (not sure what it is called)
* isEmpty() instead of size() > 0
* removed unused private variable

### CODE REVIEW

File: src/main/java/net/sf/jabref/gui/ContentSelectorDialog2.java [hunk 1]
```diff
@@ -268,12 +268,12 @@ private void applyChanges() {
         }
 
         // Cycle through all fields that we have created listmodels for:
-        for (String fieldName : wordListModels.keySet()) {
+        for (Map.Entry<String, DefaultListModel<String>> entry : wordListModels.entrySet()) {
```
Comment (Siedlerchr) [inline src/main/java/net/sf/jabref/gui/ContentSelectorDialog2.java]: Just an idea: Maps etc support Streams with Lambdas, too.
http://www.mkyong.com/java8/java-8-filter-a-map-examples/

**Comment by Siedlerchr** [comment]
LGTM :+1:

File: src/main/java/net/sf/jabref/gui/ContentSelectorDialog2.java (hunk 1 above)
Comment (oscargus) [inline src/main/java/net/sf/jabref/gui/ContentSelectorDialog2.java]: Out of my league at the moment, but most likely a good idea! :-)

**Comment by stefan-kolb** [comment]
LGTM :+1:

**Comment by tobiasdiez** [comment]
LGTM :+1:

**Pull request merged**